editor/src/clj/editor/lsp/server.clj

```diff
@@ -67,13 +68,21 @@
 (extend-protocol Launcher
   Map
   (launch [{:keys [command]} directory]
-    (.start
-      (doto (ProcessBuilder. ^"[Ljava.lang.String;" (into-array String command))
-        (.redirectError ProcessBuilder$Redirect/INHERIT)
-        (.directory directory)))))
+    {:pre [(vector? command)]}
+    ;; We need to resolve command in addition to setting ProcessBuilder's directory,
+    ;; since the latter only affects the current directory of the spawned process,
+    ;; and does not affect executable resolution
+    (let [resolved-command (update command 0 #(workspace/to-absolute-path (str directory) %))]
```

Comment:
Are you aware `workspace/to-absolute-path` doesn't actually return an absolute path? It returns a project-relative path from a resource-relative path. I hate this.

Reply:
What? It returns an absolute path if the supplied path is relative, and the original path if the supplied path is absolute, which is what we want here:
```clj
(to-absolute-path "/path/to/project/root" "bin/foo.sh")
=> "/path/to/project/root/bin/foo.sh"
(to-absolute-path "/path/to/project/root" "/bin/foo.sh")
=> "/bin/foo.sh"
```